src/components/input/__tests__/input-test.js

```diff
@@ -0,0 +1,15 @@
+/** @jsx React.DOM */
+
+jest.dontMock('../code/views/inputView.jsx');
+
+var React = require('react/addons');
+var InputComponent = require('../code/views/inputView.jsx');
+var TestUtils = React.addons.TestUtils;
+
+describe('InputComponent', function() {
```

Comment:
I see you've added a couple tests, there is still a lot more that needs to be tested.

Here are the tests that I would expect at a minimum:
- When the component is in an error state
  - is the error class present
  - is there an error span
- When the component is disabled
  - is the disabled class present/is the element disabled
- When the component is provided with a label
  - is the label displayed

I'd also add tests for the `onChange` function testing every difference scenario.